spec/initializers/feature_toggles_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe 'config/initializers/feature_toggles.rb' do
+  let(:user) { build(:user) }
+
+  around do |example|
+    original = ENV['BETA_TESTERS']
+    example.run
+    ENV['BETA_TESTERS'] = original
+  end
+
+  context 'when beta_testers is ["all"]' do
+    before { ENV['BETA_TESTERS'] = 'all' }
+
+    it 'returns true' do
+      require './config/initializers/feature_toggles' # execute the initializer's code block
+
+      enabled = OpenFoodNetwork::FeatureToggle.enabled?(:customer_balance, user)
+      expect(enabled).to eq(true)
+    end
+  end
+
+  context 'when beta_testers is a list of emails' do
+    context 'and the user is in the list' do
+      let(:other_user) { build(:user) }
+      before { ENV['BETA_TESTERS'] = "#{user.email}, #{other_user.email}" }
+
+      it 'enables the feature' do
+        require './config/initializers/feature_toggles' # execute the initializer's code block
+
+        enabled = OpenFoodNetwork::FeatureToggle.enabled?(:customer_balance, user)
+        expect(enabled).to eq(true)
+      end
+    end
+
+    context 'and the user is not in the list' do
+      before { ENV['BETA_TESTERS'] = '' }
```

Comment:
It's a maybe bit exaggerated, but we could check if the list is empty, and if the list contains a user like: 

```suggestion
      before { ENV['BETA_TESTERS'] = "#{other_user.email}" }
```
But i guess, it's almost the same...

Reply:
yes, I also thought having the list empty would be much more clear for the reader but your suggestion is good too.

Reply:
Yes, that's what I wanted to say!